package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "linter-gjslint",
   "main": "./lib/init",
-  "version": "1.0.1",
+  "version": "1.1.0",
```

Comment:
I made the same mistake when I first started working on Atom packages, don't feel bad :wink:.